Makefile

```diff
@@ -54,6 +63,16 @@ install-clippy:
 install-wasm32:
 	rustup target add wasm32-unknown-unknown
 
+setup-http-server: certs $(CERT_OPTS)
+	cargo install http-server
```

Comment:
Can you try using [bininstal](https://github.com/cargo-bins/cargo-binstall)?  This will avoid compiling and speed up CI